src/zui/components/ZUIVerticalBarChart/ZUIVerticalBarChartScale.tsx

```diff
@@ -0,0 +1,71 @@
+import { Box, Typography } from '@mui/material';
+import React, { FC } from 'react';
+import { useTheme } from '@mui/system';
+
+export interface ZUIVerticalBarChartScaleProps {
+  maxValue: number;
+  minValue?: number;
+  orientation?: 'horizontal' | 'vertical';
+}
+const ZUIVerticalBarChartScale: FC<ZUIVerticalBarChartScaleProps> = ({
+  maxValue,
+  minValue = 0,
+  orientation = 'vertical',
+}) => {
+  const theme = useTheme();
+  const style = {
+    line: {
+      backgroundColor: 'currentColor',
+      height: '1px',
+      width: theme.spacing(0.5),
+    },
+    lines: {
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'space-between',
+    },
+    numbers: {
+      '& span:first-of-type': {
+        translate: '0 -50%',
+      },
+      '& span:last-of-type': {
+        translate: '0 50%',
+      },
+      display: 'flex',
+      flexDirection: 'column',
+      justifyContent: 'space-between',
+      textAlign: 'right',
+    },
+    root: {},
+    scale: {
+      color: 'text.secondary',
+      display: 'flex',
+      gap: 0.5,
+      height: '100%',
+      justifyContent: 'space-between',
+      marginInlineStart: orientation === 'horizontal' ? '20%' : 0,
+    },
+  };
+
+  return (
+    <Box aria-hidden="true" className="scaleContainer" sx={style.root}>
```

Comment:
When you have moved the styles to be written into the `sx` of each component, you can remove the `className` property on each of these components :)